viewer/apiUsers.js

```diff
@@ -37,6 +37,12 @@ class UserAPIs {
     return user.columnConfigs || [];
   }
 
+  // --------------------------------------------------------------------------
+  static #userInfoFields (user) {
```

Comment:
this method only used in one place and couldn't it just be `user.?infoFieldsConfigs ?? []` :)